packages/contracts/src/networks.ts

```diff
@@ -722,43 +759,26 @@ export const SPHINX_NETWORKS: Array<SupportedNetwork> = [
     decimals: 18,
     queryFilterBlockLimit: 2000,
     legacyTx: false,
+    actionGasLimitBuffer: false,
+  },
+  {
+    name: 'rootstock',
+    chainId: BigInt(30),
+    rpcUrl: () => process.env.ROOTSTOCK_RPC_URL!,
+    etherscan: {
+      apiURL: 'https://rootstock.blockscout.com/api',
+      browserURL: 'https://rootstock.blockscout.com/',
+      envKey: 'ROOTSTOCK_ETHERSCAN_API_KEY',
+      blockExplorer: 'Blockscout',
+    },
+    currency: 'RBTC',
+    dripSize: '0.001',
+    requiredEnvVariables: ['ROOTSTOCK_RPC_URL'],
```

Comment:
General question: does the `requiredEnvVariables` field not need to include the etherscan api key?

Reply:
The pattern is intended to support that use case, but I haven't implemented any of the logic for it. Even if we included the Etherscan API key in this array though, we'd still need to have the field defined separately in `etherscan.envKey` because we need to know specifically what env variable to read from to get the key.